pom.xml

```diff
@@ -406,6 +406,36 @@
               </execution>
             </executions>
           </plugin>
+         <plugin>
```

Comment:
@XenoAmess 
The PR looks good but I am wondering why we need to configure the Javadoc plugin here. If there is something missing in the parent POM, we should fix it there. What is this doing for you that the parent is not?

Reply:
> @XenoAmess
> The PR looks good but I am wondering why we need to configure the Javadoc plugin here. If there is something missing in the parent POM, we should fix it there. What is this doing for you that the parent is not?

@garydgregory 
Yes I do have a reason to do so.

to repreduce what I met, please follow the steps below:
(my environment win10x64, openjdk13, but I don't think environment will matter much in this problem)

1. clone this pr in your pc.
2. see what I changed in this line: https://github.com/apache/commons-email/pull/16/files/6a9fe4a9ff73a1119c3624f8d861b01d7c6715fd#diff-77fad0477b22b44d8e52db08a61ff77eL63
3. please revert it back to the line content which before I commit the change,
means `    /** pattern for extracting <img> tags */` .
4. run `mvn package` for this repo.
5. see the build error reported.

6. please go delete the maven-javadoc-plugin in pom.xml.
7. run `mvn package` for this repo.
8. see the build success reported.

So for conclusion, I add this plugin here in this pom to [force javadoc be correct], by forcing maven fail if [javadoc be uncorrect].

I have no idea whether we can change something in commons-parent to have a same outcome without adding javadoc plugin in this repo. I'm not that familiar with complex maven management.